src/media/js/main.js

```diff
@@ -151,7 +151,9 @@ function(_) {
         // are in a packaged app, wait for the iframe to be loaded, otherwise
         // we are using the direct installer and we just need to wait for the
         // normal loaded event.
-        var event_for_apps = window.location.protocol === 'app:' ? 'iframe-install-loaded' : 'loaded';
+        var is_packaged_app = window.location.protocol === 'app:';
```

Comment:
basta used to use `u_n_d_e_r_s_c_o_r_e_s`, but we've since been trying to `camelCaseVariables`.